docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -67,6 +78,16 @@ applies the update to nodes according to the `UpdateConfig` policy:
     redis
     ```
 
+    The scheduler applies rolling updates as follows:
+
+    * Stop the initial number of tasks according to `--update-parallelism`.
+    * Schedule updates for the stopped tasks.
+    * Start the containers for the updated tasks.
+    * Wait for the health checks on individual tasks to return a state of either
```

Comment:
Please let me know if this needs more updates. If not, let's move forward and merge.